jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/GradleProjectProperties.java

```diff
@@ -134,10 +132,7 @@ private static boolean isProgressFooterEnabled(Project project) {
 
   @Override
   public JibContainerBuilder createContainerBuilder(
```

Comment:
Hmm... yeah, it feels odd. It does make sense to pass necessary initialize arguments instead of passing `JavaContainerBuilder` as before. But at the same time, I don't want to pass too many arguments. At least I'll rename it to `createJibContainerBuilder`. If we end up disliking it, we can go back.